### DISCUSSION TITLE
Report based on Activity

### DISCUSSION THREAD

**Discussion by ddeimeke** [kimai/kimai#3068] [Questions & Answers] [answered]
Hi!

I am pretty new to Kimai 2, what a great piece of software. Thank you!

There is one thing I am missing on the report side of things (or maybe I did not find it yet).

Is it possible to generate an "activities report".

There are a couple of activities I use in several projects like "Research" or "Podcasting" or "Support" and I would like to have an idea how much time I spend on such activities.

Thanks a lot for any hint.

Cheers

Dirk

**Comment by kevinpapst** [comment, marked as answer]
Hi Dirk!
Reporting on activities is not yet there. Time will fix that for sure ... or do you want to work on a PR? 😄 
You can open a feature request for that and please add some descriptions: eg. which data for global and which data for project based activities should be calculated/displayed.

**Reply by ddeimeke** [reply]
Hi Kevin!

I think a pull request with my skills would not be the kind of collaboration you want to have. :-)

But for sure, I will open a feature request and I am willing to test it on my instance.

Thank you

Dirk
